error-handler.js

```diff
@@ -1,3 +1,6 @@
 exports.errorHandler = (err, req, res, next) => {
+  if (err.status === 404) {
+    res.status(404).send(err.msg);
+  }
   res.status(400).send(err.message);
```

Comment:
If you're sending error objects that have a status and message key then you may as well make a more dynamic response line that uses both of those and means you don't need as many response lines in this errorHandler >> the customError notes cover this

Why are you sending a res.status(400) - what test is this fulfilling?

Reply:
The 400 was my default message from before on the error handler.  I've made the error handler more dynamic without the if statement and hardcoding the status